mobify/documentation-theme
# Add bullet-head and label components

## How to Test

* In this repo, run (if you haven't before) `npm link`
* Clone the documentation-hub repository and switch to [this branch](https://github.com/mobify/documentation-hub/pull/207).
* Run `npm link @mobify/documentation-theme`
* Run `npm start`
* Visit http://localhost:9000/platform/release-notes/2019-march/
* **Verify** that the Label and BulletHead styles work as intended (see screenshot below)

> **Note**: Ignore content in the PR posted above. Just focus on the design and styles!

| New Component Styles for `c-label` and `c-bullet-head` |
| --- |
| ![Screen Shot 2019-03-28 at 13 00 39](https://user-images.githubusercontent.com/734535/55188867-95486680-5159-11e9-8e7a-2a1e3e33ab8a.png) |
